#################################
# PID parameters for pose seeking
#################################

# Maximum value of error integral for all PID controllers
windup_limit: 0.05

# PID gains
x_proportional_gain: 100.0
y_proportional_gain: 100.0
z_proportional_gain: 100.0
x_integral_gain: 10.0
y_integral_gain: 10.0
z_integral_gain: 10.0
x_derivative_gain: 10.0
y_derivative_gain: 10.0
z_derivative_gain: 10.0

angular_proportional_gain: 50.0
angular_integral_gain: 5.0
angular_derivative_gain: 5.0
